### ISSUE TITLE
Additional paranthesis in PiranhaJava

### ISSUE THREAD

**Issue by mkr-plse** [uber/piranha#35]
Given input:
```
if(cond1 || (cond2 && experiments.isToggleEnabled(STALE_FLAG)) {
  // do something
}
``` 
the output of PiranhaJava is 

```
if(cond1 || (cond2)) {
  // do something
}
```

The additional paranthesis around `cond2` should be eliminated.

**Comment by jitendrs** [comment]
can you please assign to me?

**Comment by ketkarameya** [comment]
This issue has been addressed in the latest release (0.1.6). Maybe we can close this issue?

**Comment by mkr-plse** [comment]
Thanks @ketkarameya. Closing this issue.
